backend-rust/src/indexer.rs

```diff
@@ -4578,11 +4593,13 @@ impl PreparedPaydayLotteryPowers {
                     )
                     .map_err(RPCError::ParseError)?,
             );
+            ranks.push((capacity - rank) as i64);
```

Comment:
It seems to expect the bakers in ascending order here, so maybe the comment above the `sorted_by` is off